commons-core/src/main/scala/com/avsystem/commons/misc/Opt.scala

```diff
@@ -140,6 +140,9 @@ final class Opt[+A] private(private val rawValue: Any) extends AnyVal with Seria
   @inline def toLeft[X](right: => X): Either[A, X] =
     if (isEmpty) Right(right) else Left(value)
 
+  @inline def zip[B](that: Opt[B]): Opt[(A, B)] =
```

Comment:
For consistency, the same should be done for `NOpt` and `OptRef`

Reply:
IMHO, creating two lambdas for such a simple operation is a bit overkill. Can we do it in more "manually", e.g. similarly to how Scala 2.13 `Option` is doing it?